src/js/views/maps/LayerCategoryListView.js

```diff
@@ -42,36 +48,41 @@ define([
 
       /**
        * The array of layer categories to display in the list
-       * @type {LayerCategoryItemView[]}
+       * @type {ExpansionPanel[]}
        */
-      layerCategoryItemViews: undefined,
+      panels: undefined,
```

Comment:
maybe:

```suggestion
      panels: [],
```

to prevent errors (e.g. with `_.reduce`) if `options.collection` isn't passed to the view and then `view.panels` is undefined

Reply:
Done